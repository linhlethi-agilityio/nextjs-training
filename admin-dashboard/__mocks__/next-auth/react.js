export const useSession = () => ({
  data: {
    user: {
      token: 'access-token',
      email: 'user@gmail.com',
      id: '1',
      name: 'User',
      role: 'admin',
    },
    expires: Date.now().toString(),
  },
  status: 'authenticated',
  update: async () => null,
});
