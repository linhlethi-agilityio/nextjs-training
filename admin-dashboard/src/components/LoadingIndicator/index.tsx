import { Box, Spinner as SpinnerChakraUI } from '@chakra-ui/react';

const LoadingIndicator = (): JSX.Element => {
  return (
    <Box>
      <SpinnerChakraUI
        size="lg"
        thickness="4px"
        speed="0.65s"
        color="brand.500"
      />
    </Box>
  );
};

export default LoadingIndicator;
