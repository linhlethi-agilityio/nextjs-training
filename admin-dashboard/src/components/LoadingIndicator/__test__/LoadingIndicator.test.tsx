import { render } from '@testing-library/react';

// Components
import LoadingIndicator from '..';

describe('LoadingIndicator', () => {
  test('should match snapshot for Table', () => {
    const { container } = render(<LoadingIndicator />);

    expect(container).toMatchSnapshot();
  });
});
