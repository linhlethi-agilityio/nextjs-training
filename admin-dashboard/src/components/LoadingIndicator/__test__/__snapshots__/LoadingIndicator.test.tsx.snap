// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIndicator should match snapshot for Table 1`] = `
<div>
  <div
    class="css-0"
  >
    <div
      class="chakra-spinner css-1ll44wm"
    >
      <span
        class="css-8b45rq"
      >
        Loading...
      </span>
    </div>
  </div>
</div>
`;
